### ISSUE TITLE
Mobile versions in product details not updated when Fenix rc builds are shipped to release

### ISSUE THREAD

**Issue by l-hedgehog** [mozilla-releng/shipit#685]
https://product-details.mozilla.org/1.0/mobile_versions.json still shows `86.1.1` as latest version.

@jcristau [mentioned in slack](https://mozilla.slack.com/archives/CKM7DLL67/p1616747998130600?thread_ts=1616691879.121100&cid=CKM7DLL67) this may be the result of `87.0.0-rc.1` not passing the `is_release` check [here](https://github.com/mozilla-releng/shipit/blob/production/api/src/shipit_api/admin/product_details.py#L870).

**Comment by jcristau** [comment]
The versioning scheme has changed so this shouldn't be happening anymore.  Can you confirm @l-hedgehog ?

**Comment by l-hedgehog** [comment]
> The versioning scheme has changed so this shouldn't be happening anymore. Can you confirm @l-hedgehog ?

Yes, I can confirm [87.0.0-rc.1](https://github.com/mozilla-mobile/fenix/releases/tag/v87.0.0-rc.1) is the last release with such a version, and we've never experienced the same issue in product details since then.
